src/Nancy/HttpLinkBuilder.cs

```diff
@@ -0,0 +1,49 @@
+namespace Nancy
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Builds an RFC 5899 Link HTTP header as a <see cref="List{T}" /> of <see cref="HttpLink" /> objects.
+    /// </summary>
+    /// <seealso cref="HttpLink" />
+    public class HttpLinkBuilder : List<HttpLink>
```

Comment:
Here I think it most definitely make sense. A link builder (and the header it's outputting to) **is** an enumeration of links. I think this is very intuitive and makes the builder easy to understand and use, since it implements a well-known interface.